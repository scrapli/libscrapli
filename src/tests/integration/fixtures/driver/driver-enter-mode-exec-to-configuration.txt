Warning: Permanently added '[localhost]:22022' (ED25519) to the list of known hosts.
(admin@localhost) Password: 
Last login: Thu Aug 28 13:31:09 2025 from 140.82.116.3
[5neos1>
eos1>
eos1>enable
Password: 
eos1#configure terminal
eos1(config)#