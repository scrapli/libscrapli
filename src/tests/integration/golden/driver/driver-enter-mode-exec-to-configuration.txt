eos1>
eos1>enable
eos1#
eos1#configure terminal
eos1(config)#